Clients/GUI/Authentication/AuthStateProvider.cs

```diff
@@ -10,20 +11,21 @@ namespace GUI.Authentication
     /// </summary>
     public class AuthStateProvider : AuthenticationStateProvider
     {
-        private readonly ILocalStorageService localStorage;
```

Comment:
Why are you changing to this?

Reply:
> Why are you changing to this?

Because I decided that right now we don't want data on client side to be able to live that long as local storage makes it possible, so it's kind of overkill. If we will need local storage over session storage later, it can be changed in 1 minute